pkg/controller/pkg/manager/reconciler.go

```diff
@@ -233,7 +233,7 @@ func (r *Reconciler) Reconcile(req reconcile.Request) (reconcile.Result, error)
 
 	p.SetConditions(v1alpha1.Unpacking())
 
-	hash, err := r.digest.Fetch(ctx, p)
+	hash, err := r.digester.Digest(ctx, p)
```

Comment:
Nit: `r.package.Digest(ctx, p)` would be more in line with the typical (albeit possibly controversial) prose-like struct field names we use in other reconcilers.

Reply:
Went with `pkg` because I couldn't use `package` in the struct definition as it collides `package` special word